### PR TITLE
feat(orchestrator): change openapi client generator

### PR DESCRIPTION
**Pull request by gciavarrini**
Update the orchestrator service to leverage `openapi-generator-cli` with `typescript-axios` for OpenAPI client generation. Currently, we're using `@hey-api/openapi-ts,` which lacks the ability to programmatically set the base URL path. By using a widely adopted tool like `openapi-generator-cli`, we improve code maintainability and consistency within the project.

This PR marks the first in a series of changes aimed at enabling the orchestrator service to utilize only v2 API endpoints.

### CODE REVIEW

File: plugins/orchestrator-common/package.json
Comment (gciavarrini) [inline plugins/orchestrator-common/package.json:1]: @batzionb this change is mandatory, otherwise the `git commit` fails.
Is it ok?

File: yarn.lock
Comment (gciavarrini) [inline yarn.lock:1]: @batzionb is it ok to commit this?

Comment (batzionb) [reply yarn.lock:1]: yarn.lock?
we need approval from janus maintainers

**Review by batzionb — APPROVED** [review]

**Comment by gciavarrini** [comment]
@nickboldt
There are changes into root `yarn.lock`, so I need approval from janus maintainers : can you PTAL?

**Comment by nickboldt** [comment]
/lgtm

@kim-tsao @schultzp2020 how does this look to you?

**Pull request merged**
